tests/Fixer/FunctionNotation/NullableTypeHintForDefaultNullValueFixerTest.php

```diff
@@ -0,0 +1,240 @@
+            '<?php function foo(?Baz &$param = null) {}',
+        ];
+        yield [
+            '<?php function foo(\Baz &$param = null) {}',
+            '<?php function foo(?\Baz &$param = null) {}',
+        ];
+        yield [
+            '<?php function foo(Bar\Baz &$param = null) {}',
+            '<?php function foo(?Bar\Baz &$param = null) {}',
+        ];
+        yield [
+            '<?php function foo(\Bar\Baz &$param = null) {}',
+            '<?php function foo(?\Bar\Baz &$param = null) {}',
+        ];
+        yield [
+            '<?php function foo(Baz & $param = null) {}',
+            '<?php function foo(?Baz & $param = null) {}',
+        ];
+        yield [
+            '<?php function foo(\Baz & $param = null) {}',
+            '<?php function foo(?\Baz & $param = null) {}',
+        ];
+        yield [
+            '<?php function foo(Bar\Baz & $param = null) {}',
+            '<?php function foo(?Bar\Baz & $param = null) {}',
+        ];
+        yield [
+            '<?php function foo(\Bar\Baz & $param = null) {}',
+            '<?php function foo(?\Bar\Baz & $param = null) {}',
+        ];
+        yield [
+            '<?php function foo(array &$param = null) {}',
+            '<?php function foo(?array &$param = null) {}',
+        ];
+        yield [
+            '<?php function foo(array & $param = null) {}',
+            '<?php function foo(?array & $param = null) {}',
+        ];
+        yield [
+            '<?php function foo(callable $param = null) {}',
+            '<?php function foo(?callable $param = null) {}',
+        ];
+        yield [
+            '<?php $foo = function (Baz $param = null) {};',
+            '<?php $foo = function (?Baz $param = null) {};',
+        ];
+        yield [
+            '<?php $foo = function (Baz &$param = null) {};',
+            '<?php $foo = function (?Baz &$param = null) {};',
+        ];
+        yield [
+            '<?php $foo = function (Baz & $param = null) {};',
+            '<?php $foo = function (?Baz & $param = null) {};',
+        ];
+        yield [
+            '<?php class Test { public function foo(Bar\Baz $param = null) {} }',
+            '<?php class Test { public function foo(?Bar\Baz $param = null) {} }',
+        ];
+        yield [
+            '<?php class Test { public function foo(self $param = null) {} }',
+            '<?php class Test { public function foo(?self $param = null) {} }',
+        ];
+
+        yield [
+            '<?php function foo(iterable $param = null) {}',
+            '<?php function foo(?iterable $param = null) {}',
+        ];
+        yield [
+            '<?php $foo = function(array $a = null,
+                    array $b = null, array     $c = null, array
+                    $d = null) {};',
+            '<?php $foo = function(?array $a = null,
+                    ?array $b = null, ?array     $c = null, ?array
+                    $d = null) {};',
+        ];
```

Comment:
Please add test cases with comments and/or whitespaces between `?` and the type.

Reply:
Done